Task2_decorator.py

```diff
@@ -0,0 +1,24 @@
+# Task2
+string = input()
```

Comment:
do not use global variables to pass object inside the functions, this is what parameters are for